ansible/roles/satellite-manage-manifest/tasks/main.yml

```diff
@@ -1,8 +1,8 @@
 ---
 
 ## Import for version satellite 6.4 ##
-- import_tasks: version_6.4.yml
-  when: satellite_version == 6.4
+- import_tasks: version_6.x.yml
+  when: satellite_version >= 6.4
```

Comment:
This won't work. Ansible will convert to float. For example:

`6.10 >= 6.4` is false

There is a special jinja2 filter for that.

```
when: satellite_version is version_compare('6.4', '>=')
```
You test work